packages/ia-topnav/src/wayback-search.js

```diff
@@ -0,0 +1,61 @@
+import { html } from 'lit-element';
+import TrackedElement from './tracked-element';
+import waybackSearchCSS from './styles/wayback-search';
+import icons from './assets/img/icons';
+
+class WaybackSearch extends TrackedElement {
+  static get styles() {
+    return waybackSearchCSS;
+  }
+
+  static get properties() {
+    return {
+      config: { type: Object },
+      locationHandler: { type: Function },
+    };
+  }
+
+  constructor() {
+    super();
+    this.config = {};
+    this.locationHandler = () => {};
+  }
+
+  handleSubmit(e) {
+    e.preventDefault();
+    const query = e.target.querySelector('#url').value;
+    this.locationHandler(`https://${this.config.waybackUrl}/web/*/${query}`);
```

Comment:
When I think of a "handler" object, I tend to think of it as one that can manage a whole category of some functionality vs a single function. In this case, a good parallel would be the `window.history` handler. It manages all of the history functionality ie `forward()`, `back()`, `pushState()`, etc. 

It might just be a naming difference, but I think "handler" has a much broader/vague meaning in this context. Usually when you call a function, there's a verb involved (ie `setLocation()`, `pushState()`) vs `this.locationHandler('...')`. By just reading the name of that call, it doesn't really tell me the action that's going to be taken.

I think the second part of this is passing a function in verb form into your component/object reads kind of weird:
```
<wayback-search .setLocation=${this.setLocation}> // kind of odd here

// wayback-search.js
this.setLocation('some_url'); // reads okay here
```
vs
```
<wayback-search .locationHandler=${this.locationHandler}>

// wayback-search.js
this.locationHandler.setLocation('some_url');
```